### ISSUE TITLE
SLF4J Warning after updating to 1.20

### ISSUE THREAD

**Issue by adepalle** [detekt/detekt#4811]
Hi,

I know an issue was already created on this problem during 1.20-RC2 (and was fixed) but i have this problem actually after upgrading from 1.19 to 1.20

## Expected Behavior
I saw on the old issue this warning should not be displayed

## Observed Behavior
I got this warning when lint + autoCorrect : 
`SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.`

## Steps to Reproduce
Run lint with 1.20 with autoCorrect (CLI)

## Context
I just upgraded from 1.19 to 1.20 

## Your Environment
* Version of detekt used: 1.20
* Version of Gradle used (if applicable): 7.2

**Comment by cortinico** [comment]
I believe this was addressed by #4775

**Comment by adepalle** [comment]
Thanks for your answer.

I need to wait for the version 1.21.0 so ?

**Comment by cortinico** [comment]
> Thanks for your answer.
> I need to wait for the version 1.21.0 so ?

Yes. You could technically add a no-op dependency in your configuration as well, but I'm not sure that's going to work.

**Comment by eygraber** [comment]
I'm testing out the SNAPSHOT, and I'm seeing:

```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/eli/.gradle/caches/modules-2/files-2.1/io.gitlab.arturbosch.detekt/detekt-formatting/main-SNAPSHOT/e81cf605907ce92814dde5d25dfedfc0a8d5a419/detekt-formatting-main-SNAPSHOT.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/eli/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-nop/1.7.36/a3c1eb685d59414527faa93623acae311c184032/slf4j-nop-1.7.36.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.helpers.NOPLoggerFactory]
```

**Comment by cortinico** [comment]
Thanks for the heads up. We should look into a different solution for it then
